build-tasks/common.js

```diff
@@ -0,0 +1,141 @@
+const path = require('path');
+const fs = require('fs');
+const glob = require('glob');
+
+function readFile(fileName) {
+    try {
+        return fs.readFileSync(String(fileName), {encoding: 'utf8'});
+    } catch (e) {
+        console.error(`Unable to read file: ${String(fileName)}. Error code: ${e.code}`);
+        process.exit(1);
```

Comment:
I'm under the impression we should prefer `process.exitCode = 1;`: https://nodejs.org/api/process.html#process_process_exit_code. Thoughts?

Reply:
I've played a bit with this and looks like `process.exitCode = 1;` would make sense only in in some checks that are performed at the end of file like in [`validate-config`](https://github.com/Microsoft/tslint-microsoft-contrib/pull/584/files#diff-959182fc06ca4da489fed83e7951391bR60) and [`validate-documentation`](https://github.com/Microsoft/tslint-microsoft-contrib/pull/584/files#diff-410f991c8117c6f5aca2760154ec2de4R33) where script execution will be finished right after this check and nothing depends on result.

In contrast `readFile` most likely be a first call in file and other code depends on its result. If there won't be implicit termination with `process.exit(1)`  - execution will continue and user will see `TypeError: Cannot read property 'toUpperCase' of undefined` or something similar.

To keep things similar across different cases I think it is better to use `process.exit(1)` in all places.